src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImpl.java

```diff
@@ -73,18 +246,68 @@ private int computedTotalRequestEntryDetailLinesForRequestDetail(ConcurRequestEx
         }
     }
 
-    private KualiDecimal getTotalRequestFileRequestAmount(ConcurRequestExtractFile requestExtractFile) {
-        KualiDecimal detailLinesApprovedAmountSum = KualiDecimal.ZERO;
+    private KualiDecimal getTotalRequestFileTotalApprovedAmount(ConcurRequestExtractFile requestExtractFile) {
+        KualiDecimal detailLinesTotalApprovedAmountSum = KualiDecimal.ZERO;
         if ( (requestExtractFile.getRequestDetails() == null) || (requestExtractFile.getRequestDetails().isEmpty()) ) {
-            return detailLinesApprovedAmountSum;
+            return detailLinesTotalApprovedAmountSum;
```

Comment:
if might be better to just return KualiDecimal.ZERO here
Or, you could only do the addition when there are details and return detailLinesTotalApprovedAmountSum at the end